src/ibek/gen_scripts.py

```diff
@@ -28,41 +28,60 @@ def ioc_deserialize(ioc_instance_yaml: Path, definition_yaml: List[Path]) -> IOC
 
     Returns an in memory object graph of the resulting ioc instance
     """
+    all_values: Dict[str, str] = {}
+
```

Comment:
or this

Reply:
Right that was just a horrible thing I forgot to revert. ooops.